tmp/build/patch-charts.sh

```diff
@@ -252,6 +252,13 @@ function patchTemplates() {
     sed -i -e 's/\/health/\/tmp\/health/' ${HELM_DIR}/istio/charts/sidecarInjectorWebhook/templates/deployment.yaml
   fi
 
+  # add oauth-proxy default hub, tag and imagePullPolicy
+  sed -i -e '/^global:/a \
+\  oauthproxy:\
+\    hub: docker.io/openshift\
+\    tag: v1.1.0\
```

Comment:
For community, go with latest, as 1.1.0 is way outdated.

Reply:
As long as you don't actually mean use the 'latest' tag, that's definitely out.  Which version have we been testing with?

Reply:
I literally meant `latest` tag. There are only three tags available: `latest`, one month old, `v1.1.0` more than one year old and `v1.0.0`, 2 years old. I've tested my changes on `latest`.

Reply:
This was the thrust of my comment above.  Should we be using one of the images from registry.redhat.io for product?

Reply:
IMO, yes (see my comment above). It's worth to note that jaeger, for instance, is also using docker hub for product (I believe they're going to work on this).

Reply:
We should not be using latest tags, we need to be fixing them to a specific tag or a floating tag which we expect to be compatible.